docs/src/main/tut/docs/README.md

```diff
@@ -71,17 +70,17 @@ parametrization of Free programs.
 }
 ```
 
-Freestyle automatically wires all dependencies through implicit evidences that are generated so you don't have to worry about the boilerplate required to build Free based programs.
+Freestyle automatically wires all dependencies through implicit evidences that are generated, so you don't have to worry about the boilerplate required to build Free-based programs.
 
-Once you have these abstract definitions you can combine them in whichever way you want. Freestyle supports nested modules enabling onion style architectures of any arbitrary depth.
+Once you have these abstract definitions, you can combine them in whichever way you want. Freestyle supports nested modules enabling onion-style architectures of any arbitrary depth.
 
-Learn more about [modules](./core/modules) in the extended [documentation]()
```

Comment:
Is this one of the missing links you were referring to? if so removing it like you did it's fine since we are already linking to the modules section.